### ISSUE TITLE
Transform repository into `catppuccin/code-icons`

### ISSUE THREAD

**Issue by sgoudham** [catppuccin/vscode-icons#182]
### Describe the feature

Based on previous discussion, we are interested in making this repository a "dumb" one by extracting out the extension logic into [catppuccin/vscode](https://github.com/catppuccin/vscode) and renaming this repository to `catppuccin/code-icons`.

**Comment by thelooter** [comment]
Do we want to limit the repository to just code icons. I suggest going the route of providing different modules with different types of icons. That way a more general purpose iconset like fontawesome or nerdicons could develop over time

**Comment by sgoudham** [comment]
I remember having the same idea a while back but I ended up agreeing with Winston that people might just want to pull in code-icons as a submodule instead of one generic icons repository with all the assets. 

We can have separate repositories, really not looking to introduce more monorepo-style repositories into Catppuccin xD

**Comment by thelooter** [comment]
> I remember having the same idea a while back but I ended up agreeing with Winston that people might just want to pull in code-icons as a submodule instead of one generic icons repository with all the assets. 
> 

I agree, maybe an approach with multiple npm / jsr modules would make sense. I just think that having multiple repos for icons is more confusing than it helps with eliminating monorepos.

In the end I'm gonna follow the core team decision, just my insight into things

**Comment by sgoudham** [comment]
The larger idea is to add a `Contributing` page onto the Catppuccin website that would list repositories / link to different resources in one place, which would aim to boost the visibility of all the resources developers have at their disposal.

Since we're using Astro, could probably be markdown pulled from the revamped contributing guide that I'm planning on doing for ctp/ctp. That was winston's original vision to make sure we're not repeating our documentation between website/GitHub.

**Comment by sgoudham** [comment]
> This approach would also allow for far easier icon updates via renovate, as npm packages are natively supported

We're already using Renovate across a bunch of repos and the idea to enable it for the entire organisation. Renovate monorepo support is actually not mature at all. I've been fighting with it for the last few weeks over catppuccin/toolbox haha
